lib/resource_kit/pipeline/execute/token.ex

```diff
@@ -1,16 +1,29 @@
 defmodule ResourceKit.Pipeline.Execute.Token do
   @moduledoc false
 
-  use TypedStruct
   use ResourceKit.Pipeline.Token
 
   alias ResourceKit.Types
 
   alias ResourceKit.Schema.Ref
 
-  typedstruct module: Context do
-    field :root, URI.t(), enforce: true
-    field :current, URI.t(), enforce: true
+  defmodule Context do
+    @moduledoc false
+
+    use TypedStruct
+
+    typedstruct do
+      field :root, URI.t(), enforce: true
+      field :current, URI.t(), enforce: true
+      field :dynamic, atom() | pid(), enforce: true
```

Comment:
要不叫 dynamic_repo 吧，意义更准确一些
```suggestion
      field :dynamic_repo, atom() | pid(), enforce: true
```

Reply:
这个名字我是看 Ecto 那边这么命名的，你看要不要调嘛。
https://github.com/elixir-ecto/ecto/blob/4cc725a1789cf6dd03ce2d7cc4b9173740704b4b/lib/ecto/repo.ex#L264

Reply:
你发的这个是参数名字，不是存储下来的 key 的名字
https://github.com/elixir-ecto/ecto/blob/4cc725a1789cf6dd03ce2d7cc4b9173740704b4b/lib/ecto/repo.ex#L265C35-L265C47